ext/afform/core/afform.php

```diff
@@ -174,16 +174,92 @@ function afform_civicrm_managed(&$entities) {
 }
 
 /**
- * Implements hook_civicrm_caseTypes().
+ * Implements hook_civicrm_tabset().
  *
- * Generate a list of case-types.
+ * Adds afforms as contact summary tabs.
+ */
+function afform_civicrm_tabset($tabsetName, &$tabs, $context) {
+  if ($tabsetName !== 'civicrm/contact/view') {
+    return;
+  }
+  $scanner = \Civi::service('afform_scanner');
+  $weight = 111;
```

Comment:
@colemanw is there any reasoning behind this specific weight? are we not able to just get the max weight of the current tabs?

Reply:
Well, we generally want the "ChangeLog" tab to be last and that has a weight of 120, so this puts the afform tabs after core tabs but before changelog.

The default positions for tabs and blocks can be configured with the Contact Layout Editor, which I think is the right answer for anyone wanting to change it.

Reply:
Also 111 is the emergency number in NZ